kms/api-client/quickstart.py

```diff
@@ -13,32 +13,37 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 
+import os
+
 
 def run_quickstart():
     # [START kms_quickstart]
     # Imports the Google APIs client library
-    import googleapiclient.discovery
+    from google.cloud import kms_v1
 
     # Your Google Cloud Platform project ID
     project_id = 'YOUR_PROJECT_ID'
+    # [END kms_quickstart]
+    project_id = os.environ['GCLOUD_PROJECT']
```

Comment:
The idea is that the dummy variable will show up in the docs, but it will inject the ENV VAR when running tests. It's a little strange, but I wanted to make sure the project_id variable is consistent with the other languages in the docs, while still keeping the function testable